part1/courseinfo/src/App.jsx

```diff
@@ -0,0 +1,59 @@
+const Header = (props) => {
+  return (
+    <div>
+      <h1>Course: {props.course}</h1>
+    </div>
+  )
+}
+
+const Content = (props) => {
+  return (
+    <div>
+      <Part part={props.part1} exercises={exercises1}/>
+      <Part part={props.part2} exercises={exercises2}/>
+      <Part part={props.part3} exercises={exercises3}/>
+    </div>
```

Comment:
```suggestion
    <>
      <Part part={props.part1} exercises={exercises1}/>
      <Part part={props.part2} exercises={exercises2}/>
      <Part part={props.part3} exercises={exercises3}/>
    </>
```

With this one its the same with the divs. You would have got an error if you tried to return just the parts with no outer tag and thats a JSX requirement. To get around it you can use the empty tags which are equivalent to the `<React.Fragment>` tag. They dont appear in the dom when this is rendered, whereas the divs do.